shared/src/framework/exceptions/http-error/http-error.exception.ts

```diff
@@ -1,5 +1,5 @@
-import { type HttpCode } from '~/framework/http/http.js';
-import { type ValueOf } from '~/types/value-of.type.js';
+import { type HttpCode } from 'shared/src/framework/http/http.js';
+import { type ValueOf } from 'shared/src/types/value-of.type.js';
```

Comment:
There was some kind of a problem in linter that in this file it could not find HttpCode and ValueOf. I could not commit thats why i decided to change it like this and it worked.

Reply:
fixed, i think it was some kind of a problem with my code editor which kept showing me errors in these lines

Reply:
![image](https://user-images.githubusercontent.com/105710998/228659094-65d174a0-87c7-4cbd-9ea1-3d4430002e80.png)